### DISCUSSION TITLE
Add ability to get dataurl for images/assets via GraphQL

### DISCUSSION THREAD

**Discussion by andrewfairlie** [craftcms/cms#9076] [Ideas] [answered]
Many placeholder image tools (such as Next.js 11's [blur placeholder](https://nextjs.org/docs/api-reference/next/image#blurdataurl)) require the placeholder to be a Base64 data URL. With Twig you can use `dataurl(URLHERE)` to get that, and it would be great if this feature could be exposed via GraphQL also.

Ideally, we'd be able to do something like this...

```
    url
    dataurl @transform(width: 8)
```

If this already exists, and I've been blind, sorry!

**Comment by andris-sevcenko** [comment, marked as answer]
There's a discussion around this already: https://github.com/craftcms/cms/discussions/9116
